DATA/UNIVERSE/SYSTEMS/Bw10/Bw10.ini

```diff
@@ -152,70 +156,76 @@ color = 45, 45, 55 ; 50 Prozent Original
 nebulae = solar\starsphere\starsphere_Bw10.cmp
 
 [zone]
-nickname = Zone_Bw10_path_planetform1_1
-pos = -33638, 0, 1543
+nickname = Zone_Bw10_path_sds1_1
+pos = -33638, 0, 1693
 rotate = 90, -1, 180
 shape = CYLINDER
-size = 750, 2936
+size = 750, 2636
 sort = 74
 toughness = 19 ;toughness = 9
-density = 5
-repop_time = 30
+density = 3
+repop_time = 90

```

Comment:
I think to counter the problem of Harris being harrassed, it would be better to change the respawn times and percentage back to the ones of Planetform. Right now this patrol is far less likely to appear than before. And even in lower density.